src/components/util/util.js

```diff
@@ -177,3 +177,19 @@ export const getContributors = (account, instance) => {
 
   return contributors && contributors.map(({ name }) => name);
 };
+
+export const getResponseErrors = async (response) => {
+  if (!response || response === [] || response === {}) return [];
+
+  if (Array.isArray(response)) {
+    const allRejectedResponses = response.map(res => res.value.filter(i => i.status === 'rejected')).flat();
```

Comment:
Perhaps `.flatMap` will work here?

Reply:
Yes, but we do not need the parent object since its status could be "rejected" too and it will throw an error below with .json() code block.